platform/viewer/public/html-templates/script-tag.html

```diff
@@ -32,7 +32,7 @@
       console.log('OHIF Viewer rendered/updated');
     };
     window.OHIFViewer.installViewer(
-      window.config, // See `/config/default.js`
+      { config: window.config }, // See `/config/default.js`
```

Comment:
See `App.js` comment. I'm unsure of the value of this change.

Reply:
Updated, now I think its clearer.